Update Storage Component Implementation to DeveloperGuide.md

B-enguin opened the pull request:
Replace Existing StorageClassDiagram.puml with updated Class Diagram to reflect changed to the structure of the Storage Component Update style.puml to better show generics, which was previously not visible due to FontColor Update DeveloperGuide.md to include additional information related to the `Storage` component

Close #165

B-enguin committed:
Update Storage Component Implementation to DeveloperGuide.md

Replace Existing StorageClassDiagram.puml with updated Class Diagram to reflect changed to the structure of the Storage Component
Update style.puml to better show generics, which was previously not visible due to FontColor
Update DeveloperGuide.md to include additional information related to the `Storage` component

juliusgambe reviewed (commented):
From the telegram photo, there's a hollow box pointing to storage. Is that expected?

B-enguin committed:
Modify style.puml for Hidden Classes

Add styling for hidden classes using <<HIDDEN>> tag, as black BorderColor
has conflict with displaying Generics as dotted line box will not be rendered

B-enguin committed:
Merge branch 'master' into 165-update-dg-storage

Gabriel4357 reviewed (approved):
Just making minor changes LGTM

Gabriel4357 committed:
Merge branch 'master' into 165-update-dg-storage

Gabriel4357 commented:
Done
